src/github.com/getlantern/osversion/cmd/main_android.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"log"
+
+	"github.com/getlantern/osversion"
+
+	"golang.org/x/mobile/app"
+)
+
+func main() {
+	// checkNetwork runs only once when the app first loads.
+	app.Main(func(a app.App) {
+		str, err := osversion.GetString()
+		if err != nil {
+			log.Println("Error")
+		}
+		log.Println(str)
+
+		str, err = osversion.GetHumanReadable()
+		if err != nil {
+			log.Println("Error")
+		}
+		log.Println(str)
```

Comment:
Android supports GetSemanticVersion() too right? Not a big deal since it looks like this is basically demo code, but any reason to leave it out here?